### ISSUE TITLE
[Important] Search engine of doc is not working

### ISSUE THREAD

**Issue by pmaroneh** [ansys/pydpf-core#1292]
### Description of the modifications

The search in the doc is not working at all. 
![image](https://github.com/ansys/pydpf-core/assets/76440649/e435299c-23a4-407c-b5a3-f83b44059371)
![image](https://github.com/ansys/pydpf-core/assets/76440649/f024a3ae-ae01-47ff-98c8-f15ab9880af8)
![image](https://github.com/ansys/pydpf-core/assets/76440649/0d5e5462-7b1b-45b4-bfd5-7b891d622b56)

This needs to be fixed ASAP as currently nothing can be searched in the doc.

### Useful links and references

https://dpf.docs.pyansys.com/version/stable/search.html#

**Comment by ayush-kumar-423** [comment]
Thanks @pmaroneh!
@PProfizi I second Pernelle's opinion, this is critical, please prioritize this.

**Comment by PProfizi** [comment]
@pmaroneh @ayush-kumar-423 on it
It is most likely linked to this https://github.com/ansys/pydpf-core/issues/1234

**Comment by ayush-kumar-423** [comment]
Thanks @PProfizi !

**Comment by ayush-kumar-423** [comment]
@PProfizi, thanks it works but the search results seem to be cut-off. Not critical but something we should improve.
![image](https://github.com/ansys/pydpf-core/assets/97742499/fa4a9f0f-25ae-4558-9a88-430bef507918)

**Comment by PProfizi** [comment]
@ayush-kumar-423 yes indeed it is an issue. It has been noticed by @Revathyvenugopal162 who will try and take a look at the cause.

**Comment by pmaroneh** [comment]
thanks a lot @PProfizi ! This will definitely make our life easier!
